server/sonar-web/src/main/js/app/components/DefaultHelmetContainer.js

```diff
@@ -18,13 +18,21 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 import React from 'react';
+import { connect } from 'react-redux';
 import Helmet from 'react-helmet';
+import { getSettingValue } from '../../store/rootReducer';
 
-export default function DefaultHelmetContainer({ children }) {
+function DefaultHelmetContainer({ children, isSonarCloud }) {
   return (
     <div>
-      <Helmet defaultTitle="SonarQube" />
+      <Helmet defaultTitle={isSonarCloud ? 'SonarCloud' : 'SonarQube'} />
```

Comment:
`isSonarCloud` can be `null` or `{ value: 'true' }` or `{ value: 'false' }`. When it's `{ value: 'false' }`, the title is wrong.